imageprocessor/templates/base.html

```diff
@@ -21,7 +20,7 @@
   <header>
 		  <nav class="navbar navbar-expand-lg navbar-light bg-light" style="height: 60px;">
 		  <a class="navbar-brand" href="/">
-			<img src="{% static 'img/logo.png' %}" width="100" height="100" alt="Photo Tagger">
+			<img src="https://image.ibb.co/h3yDUf/logo.png" width="100" height="100" alt="Photo Tagger">
```

Comment:
What does this look like if the image is no longer available?

Reply:
If image is not available then it will replace it with alt text, which in this case is "Photo Tagger".

Reply:
Have you tried it out? I don't think that's what happens. I'd rather see the images replaced with an onerror attribute, with something like this
<img src="https://image.ibb.co/mMwlkjlkj09f/cloud.png" width="100" height="100" alt="" onerror="this.style.display='none'">

Reply:
added onerror attribute which hides the image if not found.